features/steps/recommendations_steps.py

```diff
@@ -70,11 +70,11 @@ def step_impl(context):
     assert found.text != 'batman'
     context.driver.save_screenshot('line67.png')
 
-@then(u'I will see a "product_id" with "45" in my results')
+@then(u'I will see a "product_id" with "29" in my results')
```

Comment:
I think we can change this hard-coded values to the already existing step function of list or query scenarios